### ISSUE TITLE
[Automation] More precise adjustment of the floodgate height

### ISSUE THREAD

**Issue by nialister** [ihsoft/TimberbornMods#64]
Hello, can you change the value of the floogate height step in the code from 0.5 to 0.05? Because update 6 now has this step.

**Comment by ihsoft** [comment]
Yes, it can be done. But some backward compatibility would be needed to not reset the height on the old saves.

**Comment by nialister** [comment]
As a simple solution I can suggest to create a new action like WaterRegulatorPreciseStateAction with new step.

**Comment by ihsoft** [comment]
The new scripting approach allows setting the height with `0.01` step.
